packages/payload/src/utilities/configToJSONSchema.ts

```diff
@@ -633,6 +652,111 @@ export function entityToJSONSchema(
   }
 }
 
+export function fieldsToSelectJSONSchema({ fields }: { fields: Field[] }): JSONSchema4 {
+  const schema: JSONSchema4 = {
+    type: 'object',
+    additionalProperties: false,
+    properties: {},
+  }
+
+  for (const field of fields) {
+    switch (field.type) {
+      case 'text':
+      case 'textarea':
+      case 'number':
+      case 'date':
+      case 'code':
+      case 'point':
+      case 'radio':
+      case 'richText':
+      case 'email':
+      case 'relationship':
+      case 'upload':
+      case 'select':
+      case 'checkbox':
+      case 'json':
+        schema.properties[field.name] = {
+          type: 'boolean',
+        }
+        break
```

Comment:
You wouldn't simply make this the `default` case instead of adding every field type?